service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/azure/database/CloneControlledAzureDatabaseResourceFlight.java

```diff
@@ -12,8 +12,7 @@
 import bio.terra.workspace.service.resource.controlled.model.StepRetryRulePair;
 import bio.terra.workspace.service.resource.model.CloningInstructions;
 import bio.terra.workspace.service.workspace.flight.WorkspaceFlightMapKeys;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
```

Comment:
FWIW, I think the workspaces team is trying to avoid using `.*` to import (that they favor importing each class individually)